microprofile/testing/junit5/pom.xml

```diff
@@ -35,9 +35,14 @@
 
     <dependencies>
         <dependency>
-            <groupId>io.helidon.microprofile.cdi</groupId>
-            <artifactId>helidon-microprofile-cdi</artifactId>
-            <scope>provided</scope>
+            <groupId>io.helidon.microprofile.server</groupId>
```

Comment:
The CDI module still must be declared as provided (it is a mandatory dependency).
Server then should be defined as optional

Reply:
ok